sonar-project.properties

```diff
@@ -0,0 +1,5 @@
+sonar.sources=.
```

Comment:
as this is public, should sonar be used?
That would be only working if 90poe keep using it. Which will limit the open source repo longevity I think.

Reply:
Indeed, removed sonar references